renovate.json

```diff
@@ -0,0 +1,61 @@
+{
+    "ignorePresets": [
+        "github>grafana/grafana-renovate-config//presets/automerge"
+    ],
+    "schedule": ["before 6am on the first day of the month"],
+    "packageRules": [
+      {"groupName": "Grafana packages", "matchPackageNames": ["/^@grafana//"], "enabled": false},
```

Comment:
Not sure it'll pass validation but adding `"matchDepTypes": ["dependencies"],` would mean devDeps like plugin-e2e, eslint-config, tsconfig would continue to get updates.

Reply:
https://github.com/grafana/grafana-advisor-app/pull/100/commits/ad93b28be1379908be1c7c28aa690fde427e234b